wit_dashboard/wit-dashboard.html

```diff
@@ -3225,16 +3211,8 @@ <h2>Create similarity feature</h2>
                                       horizontal-align="right"
                                       vertical-align="bottom"
                                     >
-                                      <div class="dialog-title">
-                                        ROC curve
-                                      </div>
-                                      <div>
-                                        A receiver operating characteristic
-                                        (ROC) curve plots the true positive rate
-                                        (TPR) against the false positive rate
-                                        (FPR) at various classification
-                                        thresholds.
-                                      </div>
+                                      <div class="dialog-title">ROC curve</div>
+                                      <div>[[describeRocChart()]]</div>
```

Comment:
don't see the describe of ROC AUC for the faceted case of multiclass per-label charts

Reply:
oops, thanks for spotting!